src/main/java/com/fasterxml/jackson/dataformat/xml/util/DefaultXmlPrettyPrinter.java

```diff
@@ -126,6 +147,17 @@ public void indentObjectsWith(Indenter i)
 
     public void spacesInObjectEntries(boolean b) { _spacesInObjectEntries = b; }
 
+    /**
+     * Sets custom new-line
+     * @since 2.15
+     */
+    public DefaultXmlPrettyPrinter withCustomNewLine(String newLine) {
+        if (newLine != null) {
+            _newLine = newLine;
```

Comment:
Why not set `_newLine` to `SYSTEM_DEFAULT_NEW_LINE` if `null` is passed? This would allow reverting back to system newline.

Reply:
Agreed, I don't see any problem doing so.
done!